images/spz/spz/forms/__init__.py

```diff
@@ -511,3 +511,13 @@ class LoginForm(Form):
 
     user = TextField('User', [validators.Required('User muss angegeben werden')])
     password = TextField('Passwort', [validators.Required('Passwort muss angegeben werden')])
+
+class TagVerifyForm(Form):
+    """Represents the form to verify a tag.
+    """
+    tag = TextField(
+        'Matrikelnummer oder Kürzel'
+    )
+
+    def get_tag(self):
```

Comment:
not really required (see other forms), but since you already implemented it, it's OK IMHO.